app/src/main/java/org/p2p/wallet/solend/ui/deposits/SolendDepositsFragment.kt

```diff
@@ -0,0 +1,67 @@
+package org.p2p.wallet.solend.ui.deposits
+
+import android.os.Bundle
+import android.view.View
+import androidx.core.view.isVisible
+import androidx.recyclerview.widget.LinearLayoutManager
+import org.koin.android.ext.android.inject
+import org.p2p.uikit.utils.attachAdapter
+import org.p2p.wallet.R
+import org.p2p.wallet.common.mvp.BaseMvpFragment
+import org.p2p.wallet.databinding.FragmentSolendDepositsBinding
+import org.p2p.wallet.solend.model.SolendDepositToken
+import org.p2p.wallet.solend.ui.deposits.adapter.DepositClickListener
+import org.p2p.wallet.solend.ui.deposits.adapter.SolendDepositsAdapter
+import org.p2p.wallet.utils.popBackStack
+import org.p2p.wallet.utils.unsafeLazy
+import org.p2p.wallet.utils.viewbinding.viewBinding
+
+class SolendDepositsFragment :
```

Comment:
I would specify that it's user deposits, and not ALL Solend deposits
How about SolendUserDepositsFragment?